src/js/lib/jquery.kazoosdk.js

```diff
@@ -346,6 +346,10 @@
 			qubicleReports: {
 				'get': { verb: 'POST', url: 'accounts/{accountId}/qubicle_reports' }
 			},
+			qubicleDashboards: {
```

Comment:
This new object should follow the existing, (loose) naming convention of SDK methods, like so:
```javascript
			dashboards: {
				'getQueueDetails': { verb: 'GET', url: 'accounts/{accountId}/dashboards/queue_details/{queueId}' },
				'getQueueOverview': { verb: 'GET', url: 'accounts/{accountId}/dashboards/queue_overview' }
			},
```
This allows us to "guess" the resource's format by knowing the endpoint without having to look at the SDK.

Also, please keep the alphabetical order of those methods once you've renamed it.